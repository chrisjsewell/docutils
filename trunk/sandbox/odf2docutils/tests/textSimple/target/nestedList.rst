* First level

  * Second level

* First again

  * Second again

  * Once more

    * Third level

      * And number four

  * Closing on second level
