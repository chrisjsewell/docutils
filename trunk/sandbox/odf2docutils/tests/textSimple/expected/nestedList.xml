<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE document PUBLIC "+//IDN docutils.sourceforge.net//DTD Docutils Generic//EN//XML" "http://docutils.sourceforge.net/docs/ref/docutils.dtd">
<document source="input/nestedList.odt"><bullet_list bullet="*"><list_item><paragraph>First level</paragraph><bullet_list bullet="*"><list_item><paragraph>Second level</paragraph></list_item></bullet_list></list_item><list_item><paragraph>First again</paragraph><bullet_list bullet="*"><list_item><paragraph>Second again</paragraph></list_item><list_item><paragraph>Once more</paragraph><bullet_list bullet="*"><list_item><paragraph>Third level</paragraph><bullet_list bullet="*"><list_item><paragraph>And number four</paragraph></list_item></bullet_list></list_item></bullet_list></list_item><list_item><paragraph>Closing on second level</paragraph></list_item></bullet_list></list_item></bullet_list></document>
